roiextractors/extractors/schnitzerextractor/cnmfesegmentationextractor.py

```diff
@@ -56,8 +59,11 @@ def _tot_exptime_extractor_read(self):
         return self._dataset_file[self._group0[0]]['time']['totalTime'][0][0]
 
     def _summary_image_read(self):
```

Comment:
The name of this function does not suggest that the correlation image is being read

Reply:
this is a general method name that I have used across classes to read either mean or correlation image